scripts/generate-font-metadata.py

```diff
@@ -186,7 +187,7 @@ def anchors(self):
             for anchor in char.anchorPoints:
                 anchor_name = anchor[0]
                 if anchor_name in SbmuflFont.valid_anchor_names:
-                    x, y = ((value / self.font.em)
+                    x, y = (float((decimal.Decimal(str(value)) / decimal.Decimal(self.font.em)))
```

Comment:
Two of these floats (the one here and the one on line 232) are coming from FontForge. Options:

- Convert to string
- Use `Decimal.from_float`

I think the latter would be more accurate but still possibly architecture-dependent? No strong preference.

Reply:
Alternative with `Decimal.from_float`: #6

Reply:
If neither of these two methods are reproducible for you, play around with `normalize` and `quantize` to see if you can get a more reproducible result.